app/views/directives/notifications/notification-footer.html

```diff
@@ -0,0 +1,8 @@
+<a
+  ng-if="$ctrl.customScope.hasUnread(notificationGroup)"
```

Comment:
Would rather calculate this only when things change instead of in each digest loop.